### PR TITLE
move created modules into right place

### PR DESCRIPTION
**Pull request by svogl**
the modules should go into /lib/modules/<kernel version>/updates in order to override the kernel modules.
Did not dig into your backports port, this is just a quick fix that works on our ARM boards (imx6/7 with kernel v4.9).

### CODE REVIEW

**Comment by borkra** [comment]
Well, the intended operation here is for you to change kernel defconfig and not compile native set of modules (as outlined in README). We are not supporting having multiple set of modules on the system (native and backported).

**Comment by borkra** [comment]
It's better build fail outright, then unknown mix of drivers installed on the system. And god knows what is used. Supported configuration is: native set of Wi-Fi and Bluetooth drivers supposed to be disabled in kernel config and backported is used.

By this reason I am closing this request.

**Comment by svogl** [comment]
Well, IMHO it would have been better to follow the original concept as the approach taken provides problems with a) user-space kernel helpers (depmod &co override the kernel-provided modules) and b) at (yocto-based) image-creation time as you cannot easily include 'kernel-modules' as a meta-dependency (bluetooth, hci-uart drop out of the list) and need to manually specify dependent modules like hci-uart.

**Comment by borkra** [comment]
And here is the problem: module hci-uart and other modules has to come from the Backports package we provide not from your original kernel, if you would like to maintain our Bluetooth SIG certification. 
You can definitely choose to create you own configuration, but in this case you will have to perform the certifications yourself.

In addition a lot of users create fully or partially monolithic kernels, it's startup times a lot faster, in this case drivers cannot be overwritten through backports, no matter which directory they placed in.

Bottom line this forces the issue of configuring system properly. 

If you need help configuring your kernel please contact our technical support they do help people configuring it all the time.

**Comment by borkra** [comment]
Forgot to mention, this patch with moving files is incorrect, your objective could be accomplished by removing KMODDIR=. parameter.
